app/app.js

```diff
@@ -1363,13 +1376,10 @@ var AppComponent = React.createClass({
 
         body = (
           <div>
-            <p>
-              {usrMessages.ERR_GENERIC}
-              <a href="/">refresh your browser</a>
```

Comment:
I believe so. We should only ask users to do something that we know should work. We tell them to refresh when prob 90% of the time they will get the exact same thing - the app in a broken state